src/Button.js

```diff
@@ -30,8 +41,10 @@ class AriaMenuButtonButton extends React.Component {
 
     const ambManager = this.context.ambManager;
 
+    const {focusGroupOptions: {keybindings: {next: {key}}}} = this.props;
```

Comment:
Oh, I do not like this level of destructuring. Can you please change to a different expression?